src/theme/main.less

```diff
@@ -1,20 +1,62 @@
-// Separator Block
+// //Separator Block
 .block.separator {
+  top: unset;
   display: block;
   height: 54px;
   border: none;
-  margin: 32px 0;
+  margin: 50px 0;
+  clear: both;
+
+  &:not(.inner):not([role='presentation']) {
+    padding: 25px 0 !important;
+  }
+
+  &::after {
+    display: none !important;
+  }
 
   .line {
-    height: 54px;
+    width: unset;
+    height: unset;
```

Comment:
I don't think these unsets are needed. When the styles were in the theme, it was unsetting the height that was set here. If we don't set a height here, it doesn't need to be unset.